### PR TITLE
Database

### PR DESCRIPTION
**Pull request by code-driving**

### CODE REVIEW

File: db/schema/03_pins.sql [hunk 1]
```diff
@@ -6,7 +6,7 @@ CREATE TABLE pins (
   map_id INTEGER REFERENCES maps(id) ON DELETE CASCADE,
   lat FLOAT NOT NULL,
   lng FLOAT NOT NULL,
-  title VARCHAR(50),
```
Comment (ruowent) [inline db/schema/03_pins.sql]: I know we did this for testing purpose. Can you change the line back to 
title VARCHAR(50)

Thank you.

File: routes/logins.js [hunk 2]
```diff
@@ -2,9 +2,12 @@ const express = require('express');
...
 module.exports = (db) => {
-  // GET /login/:id -- Display login page
   router.get('/:id', (req, res) => {
     console.log('==> GET /login/:id -- Display login page')
   });
+  // router.get('/login/:id', (req, res) => {
+  //   req.session.user_id = req.params.id;
+  //   res.redirect('/');
+  // });
   return router;
```
Comment (ruowent) [inline routes/logins.js:12]: In the server.js file, the /logins is already specified. By default it is pointing to /logins page 
You can refer to server.js line 44

**Review by ruowent — CHANGES_REQUESTED** [review]
Please see comments

File: db/schema/03_pins.sql (hunk 1 above)
Comment (code-driving) [reply db/schema/03_pins.sql]: Sure, I will do it now

File: routes/logins.js (hunk 2 above)
Comment (code-driving) [reply routes/logins.js:12]: Ok, thanks!

Comment (code-driving) [reply routes/logins.js:12]: Do you mean this one? I don't have a logins page in server.js, just the one rendering index
app.get("/", (req, res) => {
  res.render("index");
});

**Pull request merged**